geopandas/array.py

```diff
@@ -255,6 +258,16 @@ def points_from_xy(x, y, z=None, crs=None):
     >>> gdf = geopandas.GeoDataFrame(
     ...     df, geometry=geopandas.points_from_xy(df['x'], df['y']))
 
+    Having geographic coordinates:
+
+    >>> df = pd.DataFrame({'longitude': [-140, 0, 123], 'latitude': [-65, 1, 48]})
+    >>> df
+    longitude  latitude
+    0       -140       -65
+    1          0         1
+    2        123        48
+    >>> geometry = geopandas.points_from_xy(df.longitude, df.latitude, crs=4326)
```

Comment:
```suggestion
    >>> geometry = geopandas.points_from_xy(df.longitude, df.latitude, crs="EPSG:4326")
```

(I feel like it might be good practice to be explicit, although of course 4326 is a well known number)